lte/gateway/configs/templates/mme.conf.template

```diff
@@ -92,6 +92,13 @@ MME :
          {% endfor %}
     );
 
+    # List of restricted PLMNs
```

Comment:
We should add a comment here that the maximum length for the restricted PLMN is 10